test/pricer_rule/utils.js

```diff
@@ -60,24 +62,30 @@ module.exports.createOrganization = async (accountProvider) => {
  * Creates and returns the tuple:
  *      (tokenRules, organizationAddress, token)
  */
-module.exports.createTokenEconomy = async (accountProvider) => {
+module.exports.createTokenEconomy = async (accountProvider, config = {}, eip20TokenConfig = {}) => {
   const {
     organization,
     organizationOwner,
     organizationWorker,
   } = await this.createOrganization(accountProvider);
 
-  const token = await this.createEIP20Token();
+  const eip20TokenDecimals = eip20TokenConfig.decimals || 18;
+  const createEIP20TokenConfig = {
+    symbol: eip20TokenConfig.symbol || 'OST',
+    name: eip20TokenConfig.name || 'Open Simple Token',
+    decimals: eip20TokenDecimals,
+  };
+  const token = await this.createEIP20Token(createEIP20TokenConfig);
 
   const tokenRules = await TokenRulesSpy.new();
 
-  const baseCurrencyCode = 'OST';
-
-  const conversionRate = 315;
+  const baseCurrencyCode = config.baseCurrencyCode || 'OST';
 
-  const conversionRateDecimals = 2;
+  // (For 1 OST = 1 BT)
+  const conversionRate = config.conversionRate || 100000;
```

Comment:
What is the intention to change default values for conversionRate, conversionRateDecimald, requiredPriceOracleDecimals, etc?

Reply:
The intention was to make them configurable by caller and have flexibility for testing . 

As you can in pay.js unit tests, requiredPriceOracleDecimals is being tested with multiple values(18, 5). 

Currently only requiredPriceOracleDecimals variable is being tested with different configurable values . If you think we don't need conversionRate, conversionRateDecimals as configurable values from caller, we can remove them. 
Let me know what do you think 🤔 ?

Reply:
Removed conversionRate, conversionRateDecimals as configurable parameters. It's not being utilized right now and may be over engineering at this point.